src/main.js

```diff
@@ -83,90 +176,38 @@ const startWithMBTiles = (mbtilesFile) => {
 
   mbtilesFile = path.resolve(process.cwd(), mbtilesFile);
 
-  const mbtilesStats = fs.statSync(mbtilesFile);
-  if (!mbtilesStats.isFile() || mbtilesStats.size === 0) {
-    console.log(`ERROR: Not valid MBTiles file: ${mbtilesFile}`);
-    process.exit(1);
-  }
-  const instance = new MBTiles(mbtilesFile + '?mode=ro', (err) => {
-    if (err) {
-      console.log('ERROR: Unable to open MBTiles.');
-      console.log(`Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`);
+  let tileServer = await buildServerConfig(mbtilesFile)
+    .catch((err) => {
+      console.log('ERROR: failed to configure server:', err);
       process.exit(1);
-    }
-
-    instance.getInfo((err, info) => {
-      if (err || !info) {
-        console.log('ERROR: Metadata missing in the MBTiles.');
-        console.log(
-          `Make sure ${path.basename(mbtilesFile)} is valid MBTiles.`,
-        );
-        process.exit(1);
-      }
-      const bounds = info.bounds;
-
-      const styleDir = path.resolve(
-        __dirname,
-        '../node_modules/tileserver-gl-styles/',
-      );
-
-      const config = {
-        options: {
-          paths: {
-            root: styleDir,
-            fonts: 'fonts',
-            styles: 'styles',
-            mbtiles: path.dirname(mbtilesFile),
-          },
+    })
+    .then((config) => startServer(null, config));
+
+  if (opts.watch) {
+    console.log(`[INFO] watching ${mbtilesFile} for changes...`);
+
+    chokidar
+      .watch(mbtilesFile, {
+        ignoreInitial: true,
+        // The file can be written to many times while tile generation is
```

Comment:
Typo here?
`to` insteaf of `too`.

HTH,
Matteo

Reply:
No, this is meant as "to", as in "writing to a file"